src/node_crypto.cc

```diff
@@ -1537,8 +1537,12 @@ static Local<Object> X509ToObject(Environment* env, X509* cert) {
       (void) BIO_reset(bio);
 
       BN_ULONG exponent_word = BN_get_word(rsa->e);
+# ifdef SIXTY_FOUR_BIT_LONG
       BIO_printf(bio, "0x%lx", exponent_word);
-
+# endif
+# ifdef THIRTY_TWO_BIT
```

Comment:
No, not exactly, there's a third case that I was hoping would break. I can handle it, it only happens on VMS and alpha